src/allmydata/web/root.py

```diff
@@ -63,15 +95,18 @@ def render_PUT(self, ctx):
                 return unlinked.PUTUnlinkedCHK(req, self.client)
         if t == "mkdir":
             return unlinked.PUTUnlinkedCreateDirectory(req, self.client)
-        errmsg = ("/uri accepts only PUT, PUT?t=mkdir, POST?t=upload, "
-                  "and POST?t=mkdir")
+        errmsg = (
+            "/uri accepts only PUT, PUT?t=mkdir, POST?t=upload, "
+            "and POST?t=mkdir"
+        )
```

Comment:
So, I'm willing to accept that this is an acceptable place for a native string, but I'm just in a _very_ cautious mindset re: string literals in review right now.